CMakeLists.txt

```diff
@@ -12,16 +12,21 @@ if (POLICY CMP0042)
   cmake_policy (SET CMP0042 NEW)
 endif()
 
-option ( LINK_PYTHON_LIBRARY "Link against python libraries" ON )
+if(POLICY CMP0078)
+  cmake_policy(SET CMP0078 NEW)
+endif()
+
+list (APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules)
 
-include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules/SWIG_CGAL_Macros.cmake)
+include (SWIG_CGAL_Macros)
 
 find_package(CGAL QUIET COMPONENTS ImageIO)
 find_package(SWIG REQUIRED)
 option( BUILD_PYTHON "Build Python bindings" ON )
 option( BUILD_JAVA "Build Java bindings" ON )
 #set it to OFF by default while it is not fully tested and thus officially supported
 option( BUILD_RUBY "Build Ruby bindings" OFF )
+option( BUILD_SHARED_LIBS "Build shared libraries" ON )
```

Comment:
could you explain this change?

Reply:
I remove it to only include cmake 3.13 changes.